arenadata/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM pivotaldata/centos-gpdb-dev:7-gcc6.2-llvm3.7 as base
+
+RUN rm -rf /opt/cmake* \
+    && yum remove -y cmake \
+    && yum install -y epel-release \
+    && yum install -y cmake3 \
+                    libzstd-devel \
+                    apr-util-devel \
+                    libuv-devel \
+                    perl-IPC-Run \
+                    perl-Test-Base \
+                    wget \
+    && yum clean all \
+    && ln -s /usr/bin/cmake3 /usr/bin/cmake \
+    && ln -s /usr/bin/ctest3 /usr/bin/ctest
+
+ARG sigar=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/community/x86_64/sigar-1.6.5-163.el7.x86_64.rpm
+ARG sigar_headers=https://ci.arenadata.io/artifactory/ADB/6.7.1_arenadata4/centos/7/community/x86_64/sigar-headers-1.6.5-163.el7.x86_64.rpm
+
+RUN rpm -i $sigar && rpm -i $sigar_headers && mkdir /home/gpadmin/bin_gpdb
```

Comment:
Move WORKDIR that creates `/home/gpadmin` higher this line or add mkdir option -p.